.github/ISSUE_TEMPLATE/other.yaml

```diff
@@ -0,0 +1,39 @@
+name: Other
+description: Use this for any other issues. Please do NOT create blank issues
+title: '[OTHER] write a small description here'
+body:
+- type: textarea
+  id: issuedescription
+  attributes:
+    label: What would you like to share?
+    description: Provide a clear and concise explanation of your issue.
+  validations:
+    required: true
+
+- type: dropdown
+  id: browser
+  attributes:
+    label: '✨ Browser'
+    description: 'What browser are you using?'
+    options:
+    - Google Chrome
+    - Brave
+    - Arc
+    - Safari
+    - Microsoft Edge
+    - Mozilla Firefox
+    - Other
+  validations:
+    required: true
```

Comment:
What is the relevance of a browser for Pyccel?

Reply:
Not much I reckon! This was removed in 478be206f70ce90d34cf2179c442cc48b906b327